1-js/05-data-types/02-number/article.md

```diff
@@ -1,136 +1,136 @@
-# Numbers
+# Números
 
-In modern JavaScript, there are two types of numbers:
+En JavaScript moderno, hay dos tipos de números:
 
-1. Regular numbers in JavaScript are stored in 64-bit format [IEEE-754](https://en.wikipedia.org/wiki/IEEE_754-2008_revision), also known as "double precision floating point numbers". These are numbers that we're using most of the time, and we'll talk about them in this chapter.
+1. Los números regulares en JavaScript son almacenados con el formato de 64-bit [IEEE-754](https://es.wikipedia.org/wiki/IEEE_coma_flotante), conocido como "números de double precisión de coma flotante". Estos números son los que estaremos usando la mayor parte del tiempo y hablaremos de ellos en este capítulo.
 
-2. BigInt numbers, to represent integers of arbitrary length. They are sometimes needed, because a regular number can't exceed <code>2<sup>53</sup></code> or be less than <code>-2<sup>53</sup></code>. As bigints are used in few special areas, we devote them a special chapter <info:bigint>.
+2. BigInt number, que representa enteros de largo arbitrario. A veces son necesarios porque un número regular no puede exceder <code>2<sup>53</sup></code> ni ser menor a <code>-2<sup>53</sup></code>. Como los bigints son usados un unas pocas áreas especiales, les dedicamos un capítulo especial <info:bigint>.
 
-So here we'll talk about regular numbers. Let's expand our knowledge of them.
+Aquí hablaremos de números regulares. Extendamos lo que ya sabemos de ellos.
 
-## More ways to write a number
+## Más formas de escribir un número
 
-Imagine we need to write 1 billion. The obvious way is:
+Imagina que necesitamos escribir mil millones (En inglés "1 billion"). La forma obvia es:
 
 ```js
 let billion = 1000000000;
 ```
 
-But in real life, we usually avoid writing a long string of zeroes as it's easy to mistype. Also, we are lazy. We will usually write something like `"1bn"` for a billion or `"7.3bn"` for 7 billion 300 million. The same is true for most large numbers.
+Pero en la vida real tratamos de evitar esribir una larga cadena de ceros porque es fácil tipear mal. 
```

Comment:
faltó una parte:  _Also, we are lazy. We will usually write something like `"1bn"` for a billion or `"7.3bn"` for 7 billion 300 million. The same is true for most large numbers._

```suggestion
Pero en la vida real tratamos de evitar esribir una larga cadena de ceros porque es fácil tipear mal.  Además, somos flojos. Por lo general, escribiremos algo como `"1bn''` por mil millones o `"7.3bn"` por 7 mil millones 300 millones. Lo mismo es cierto para la mayoría de los grandes números.
```

Reply:
NO.  No faltó, lo mejoré.    (agrandado yo?)

Es nuestra prerrogativa.

En la primera versión habia hecho así, despues empecé a explicar  "en inglés se hace tal" despues lo BORRÉ.
Los latinos no somos lazy
7.3bn no significa nada, no hay convenciones ni acuerdos.

Y no aporta nada a la explicación.
Es más, quizás le mande al ruso un PR para que lo saque del inglés, aunque ahí no es errado.

Reply:
¿qué hace resolve acá?  Espero no lo ponga de prepo, le cierro el PR.
A ver...